apps/passport-client/components/screens/AddScreen/JustAddScreen.tsx

```diff
@@ -26,16 +26,16 @@ export function JustAddScreen({
   const { error, pcd } = useDeserialized(request.pcd);
   const syncSettled = useIsSyncSettled();
 
-  const onAddClick = useCallback(async () => {
+  const onAddClick = useCallback(() => {
     try {
-      await dispatch({
+      dispatch({
```

Comment:
any reason to not `await` here?

Reply:
There's something a bit funky with the types on `dispatch`. The `dispatch()` function in `dispatch.ts` is async and returns a promise. The `Dispatcher` type, which is what `useDispatch()` returns, is `export type Dispatcher = (action: Action) => void;`, and so the function being called here also returns `void` and not `Promise<void>`.